apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -137,11 +143,21 @@ function ValidatorPageResult(): JSX.Element {
             'Last Epoch Rewards',
             'Voting Power',
             'Status',
+            'Current Epoch Rewards',
+            'Next Epoch Rewards',
         ];
 
+        const maxCommitteeSizeCountAttr = protocolConfig?.attributes['max_committee_members_count'];
```

Comment:
i think you can use the `useMaxCommitteeSize` hook here :pray:

Reply:
[3773f58](https://github.com/iotaledger/iota/pull/6601/commits/3773f58c7835e5c313bb14476ddaed255d85bd19)